3rd_party/ros/Micro-ROS/test.sh

```diff
@@ -18,7 +18,7 @@ cd Arduino/libraries/
 git clone -b foxy https://github.com/micro-ROS/micro_ros_arduino.git
 cd ~
 cd arduino-cli/
-export PATH=$PATH:/home/$USER/arduino-cli/bin
+export PATH=$PATH:/root/$USER/arduino-cli/bin
 arduino-cli board attach serial:///dev/ttyACM0 ardiunotopython
 arduino-cli compile --port /dev/ttyACM0 ardiunotopython
 arduino-cli upload --port /dev/ttyACM0 ardiunotopython
```

Comment:
do not thing the name of the file as test.sh is appropriate as its job is not just testing something an plays a crucial role as part of the setup.

Reply:
I agree - I was under the impression it was a placeholder.

Reply:
Renamed.